include/aws/common/common.h

```diff
@@ -223,6 +229,40 @@ static inline void aws_secure_zero(void *pBuf, size_t bufsize) {
 
 #define AWS_ZERO_STRUCT(object) memset(&object, 0, sizeof(object));
 #define AWS_ZERO_ARRAY(array) memset((void *)array, 0, sizeof(array));
+#define AWS_PTR_ADD(ptr, bytes) ((void *)(((char *)ptr) + (bytes)))
+#define AWS_PTR_SUB(ptr, bytes) ((void *)(((char *)ptr) - (bytes)))
+
+#define AWS_DOUBLY_LIST_INIT(sentinel) \
```

Comment:
I'm not super sure these belong in the public header file.

Reply:
Will move them out. Was unsure of where they should live.

Reply:
For clarity the ptr ones are fine

Reply:
Oh gotcha. Mike pointed out we already have a nice linked list header, so I will remove my `DOUBLE_LIST` macros and just use what we already have :)

Reply:
Decided to keep them since my use case for circular buffer was slightly different than `aws_linked_list`.